backend/server/mailing/mailing.go

```diff
@@ -9,12 +9,13 @@ import (
 	"github.com/mailjet/mailjet-apiv3-go"
 )
 
-var INFO_EMAIL = "info@csesoc.org.au" 
+var INFO_EMAIL = "info@csesoc.org.au"
 var SPONSORSHIP_EMAIL = "sponsorship@csesoc.org.au"
+
 const PUBLIC_KEY = "8afb96baef07230483a2a5ceca97d55d"
-const SECRET_KEY = "424ad90f25487e6be369a1cbb2a34694"
```

Comment:
Please rotate this key if you haven't already - remember anything on the internet is forever.

Reply:
@glebme you're the only one with access to the Mailjet account, could you take care of this please?

Reply:
Will do when I get home.

Reply:
Thanks @glebme , I've updated the Github Secrets with the new private key.